packages/web/src/common/store/pages/signon/sagas.js

```diff
@@ -73,14 +69,26 @@ const messages = {
   emailCheckFailed: 'Something has gone wrong, please try again later.'
 }
 
-// Users ID to filter out of the suggested artists to follow list and to follow by default
-let defaultFollowUserIds = new Set([])
-if (IS_PRODUCTION) {
-  // user id 51: official audius account
-  defaultFollowUserIds = new Set([51])
-} else if (IS_STAGING) {
-  // user id 1964: stage testing account
-  defaultFollowUserIds = new Set([1964])
+function* getDefautFollowUserIds() {
+  const { ENVIRONMENT } = yield getContext('env')
+  // Users ID to filter out of the suggested artists to follow list and to follow by default
+
+  let defaultFollowUserIds = new Set([])
+
+  switch (ENVIRONMENT) {
+    case 'production': {
+      // user id 51: official audius account
+      defaultFollowUserIds = new Set([51])
+      break
+    }
+    case 'staging': {
+      // user id 1964: stage testing account
+      defaultFollowUserIds = new Set([1964])
+      break
+    }
+  }
+
+  return defaultFollowUserIds
```

Comment:
Maybe a tradeoff in readability, but I think this would be cleaner as an expression. Also removes the need for mutating `defaultFollowUserIds`!

```suggestion
  return new Set({
    // user id 51: official audius account
    production: [51],
    // user id 1964: stage testing account
    staging: [1964],
  }[ENVIRONMENT] ?? [])
```

Reply:
dang i merged and this comment hadnt loaded :/ i agree though